Porting/updateAUTHORS.pm

```diff
@@ -189,6 +260,22 @@ sub read_commit_log {
             $self->_check_name_mailmap($committer_name_mm_canon,
                 $committer_name_real, $commit_info, "committer name");
             $self->_register_author($committer_name_mm_canon, "committer");
+            $committer_good= 1;
+        }
+        if (    $author_good
+            and $committer_good
+            and $committer_name_mm_canon ne $author_name_mm_canon)
+        {
+            $self->{who_stats}{$committer_name_mm_canon}{applied}++;
+        }
+    }
+    $self->finalize_commit_info($last_commit_info) if $last_commit_info;
+    if (!$commits_read) {
+        if ($self->{commit_range}) {
+            die "No commits in range '$self->{commit_range}'\n";
+        }
+        else {
+            die "Panic! There are no commits!\n";
```

Comment:
Well, not to me. 

If the update process updates nothing because the commit range evaluates to nothing then it does not surprise me that much and i dont necessarily expect to be told about it.

If the stats for a given commit range are empty and i get no stats I would like to be told that i am looking at no data.